internal/static/i18n/zh.yaml

```diff
@@ -0,0 +1,900 @@
+Errors:
+  Internal: 发生了内部错误
+  NoChangesFound: 没有变化
+  OriginNotAllowed: 这个"来源"是不被允许的
+  IDMissing: ID 丢失
+  ResourceOwnerMissing: 组织没有资源所有者
+  RemoveFailed: 无法移除
+  ProjectionName:
+    Invalid: 错误的映射名称
+  Assets:
+    EmptyKey: 资产的 Key 为空
+    Store:
+      NotInitialized: 资产存储未初始化
+      NotConfigured: 未配置资产存储
+    Bucket:
+      Internal: 创建桶时出现内部错误
+      AlreadyExists: 存储桶已存在
+      CreateFailed: 未创建存储桶
+      ListFailed: 无法读取存储桶
+      RemoveFailed: 未删除存储桶
+      SetPublicFailed: 无法将存储桶设置为公开
+    Object:
+      PutFailed: 未创建对象
+      GetFailed: 无法读取对象
+      NotFound: 找不到对象
+      PresignedTokenFailed: 无法创建签名令牌
+      ListFailed: 无法读取对象列表
+      RemoveFailed: 无法移除对象
+  Limit:
+    ExceedsDefault: 超出默认限制
+  Language:
+    NotParsed: 无法解析语言
+  OIDCSettings:
+    NotFound: OIDC 配置未找到
+    AlreadyExists: OIDC 配置已存在
+  SecretGenerator:
+    AlreadyExists: 秘密生成器已经存在
+    TypeMissing: 缺少秘钥生成器类型
+    NotFound: 未找到秘钥生成器
+  SMSConfig:
+    NotFound: 未找到 SMS 配置
+    AlreadyActive: SMS 配置已启用
+    AlreadyDeactivated: SMS 配置已停用
+  SMTPConfig:
+    NotFound: 未找到 SMTP 配置
+    AlreadyExists: SMTP 配置已存在
+    SenderAdressNotCustomDomain: 发件人地址必须在在实例的域名设置中验证。
+  Notification:
+    NoDomain: 未找到对应的域名
+  User:
+    NotFound: 找不到用户
+    AlreadyExists: 用户已存在
+    NotFoundOnOrg: 在所选组织中找不到用户
+    NotAllowedOrg: 用户不是所需组织的成员
+    UserIDMissing: 缺少用户 ID
+    DomainPolicyNil: 组织策略为空
+    EmailAsUsernameNotAllowed: 电子邮件不允许作为用户名
+    Invalid: 用户数据无效
+    DomainNotAllowedAsUsername: 域已保存，但无法使用
+    AlreadyInactive: 用户已处于停用状态
+    NotInactive: 用户未处于停用状态
+    CantDeactivateInitial: 处于初始状态的用户只能删除不能停用
+    ShouldBeActiveOrInitial: 用户不是处于启用的的或初始化的
+    AlreadyInitialised: 用户已经初始化
+    NotInitialised: 用户尚未初始化
+    NotLocked: 用户未锁定
+    NoChanges: 未发现任何更改
+    InitCodeNotFound: 未找到初始化验证码
+    UsernameNotChanged: 用户名未更改
+    Profile:
+      NotFound: 未找到个人资料
+      NotChanged: 个人资料未更改
+      Invalid: 个人资料数据无效
+    Email:
+      NotFound: 电子邮件没有找到
+      Invalid: 电子邮件无效
+      AlreadyVerified: 电子邮件已经过验证
+      NotChanged: 电子邮件未更改
+    Phone:
+      NotFound: 手机号码未找到
+      Invalid: 手机号码无效
+      AlreadyVerified: 手机号码已经验证
+    Address:
+      NotFound: 找不到地址
+      NotChanged: 地址没有改变
+    Machine:
+      Key:
+        NotFound: 未找到机器密钥
+    PAT:
+      NotFound: 未找到个人访问令牌
+    NotHuman: 用户必须是个人
+    NotMachine: 用户必须是技术人员
+    WrongType: 此用户类型不允许
+    NotAllowedToLink: 不允许使用外部身份提供者登录并注册用户
+    Username:
+      AlreadyExists: 用户名已被使用
+      Reserved: 用户名已被使用
+    Code:
+      Empty: 验证码为空
+      NotFound: 验证码不存在
+      Expired: 验证码已过期
+      GeneratorAlgNotSupported: 不支持的生成器算法
+    Password:
+      NotFound: 未找到密码
+      Empty: 密码为空
+      Invalid: 密码无效
+      NotSet: 用户未设置密码
+    PasswordComplexityPolicy:
+      NotFound: 未找到密码策略
+      MinLength: 密码太短
+      MinLengthNotAllowed: 给定的最小长度不被允许
+      HasLower: 密码必须包含小写
+      HasUpper: 密码必须包含大写
+      HasNumber: 密码必须包含数字
+      HasSymbol: 密码必须包含符号
+    ExternalIDP:
+      Invalid: 外部 IDP 无效
```

Comment:
In some local software, similar computer proper nouns are not translated, because users do not know what they are after translation, and it is difficult for maintenance personnel to accurately locate them.

Reply:
thanks for clarification :)